samples/interactive-tutorials/setup/setup_cleanup.js

```diff
@@ -0,0 +1,338 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
non-blocking-nit: in Node.js you tend to use `-` rather than snakecase:

```
setup-cleanup.js
```

Reply:
Done